web/tendrl/user_password_edit_dialog.rst

```diff
@@ -0,0 +1,69 @@
+Change user password at user "Edit" page
+*****************************************
+
+:authors: 
+          - mkudlej@redhat.com
+
+Description
+===========
+
+Change user password at user "Edit" page for these user types:
+ 1. *Admin* user
+ 2. *Power User* user
+ 3. *Read-only* user
+
+Design describing "Edit" page https://redhat.invisionapp.com/share/KNB25OEQT#/screens/226063805
```

Comment:
There seems to be two design documents related to this testcase, so it might be worth to mention also the second one: https://redhat.invisionapp.com/share/8QCOEVEY9#/screens/252535083

Reply:
Second link is to *add* user. I've linked design for *edit* user.